model/page.go

```diff
@@ -28,6 +28,7 @@ type Page struct {
 type FeatureFlags struct {
 	HideCookieBanner       bool   `json:"hide_cookie_banner"`
 	ONSDesignSystemVersion string `json:"ons_design_system_version"`
```

Comment:
Will likely keep this property in until the frontend services are no longer using it. Once changed, it could be removed